### PR TITLE
feat: use the new ABI for mech

### PR DESCRIPTION
**Pull request by mohandast52**
- should work the same as before
- updated old mech ABI with a new one

Screenshot 1

Screenshot 2

### CODE REVIEW

File: common-util/Contracts/index.jsx [hunk 1]
```diff
@@ -53,7 +53,7 @@ export const getMechContract = () => {
 
 export async function fetchGraphQLData() {
   return new Promise((resolve, reject) => {
-    const url = 'https://api.studio.thegraph.com/query/46780/mech/v0.0.1';
+    const url = 'https://api.studio.thegraph.com/query/57238/mech/v0.0.1';
```
Comment (kupermind) [inline common-util/Contracts/index.jsx:56]: Is this the updated query accounting for two AgentFactories?

File: components/Home/Mech/index.jsx [hunk 2]
```diff
@@ -17,10 +16,8 @@ import {
   NA,
 } from '@autonolas/frontend-library';
 
-import { NEW_MECH_ADDRESS } from 'util/constants';
 import { AGENT_MECH_ABI } from 'common-util/AbiAndAddresses';
 import { SUPPORTED_CHAINS } from 'common-util/Login';
-import { NEW_MECH_ABI } from 'common-util/AbiAndAddresses/newMech';
```
Comment (kupermind) [inline components/Home/Mech/index.jsx:23]: This file can be removed then.

**Review by kupermind — APPROVED** [review]

File: components/Home/Mech/index.jsx [hunk 3]
```diff
@@ -239,7 +234,7 @@ const EventListener = () => {
       >
         <Request />
         <Alert
-          message="We have encountered challenges with data retrieval, resulting in a temporary unavailability of older requests."
+          message={`We have encountered challenges with data retrieval, resulting in a temporary unavailability of older requests. NOTE: We only load the latest ${LATEST_BLOCK_COUNT} blocks`}
```
Comment (oaksprout) [inline components/Home/Mech/index.jsx:237]:
```suggestion
          message={`We only load the latest ${LATEST_BLOCK_COUNT} blocks due to data availability issues.`}
```

Comment (oaksprout) [reply components/Home/Mech/index.jsx:237]: @mohandast52 we should also add a message to the empty state on this list.

```
No requests
<br />
Only loading latest LATEST_BLOCK_COUNT block(s). [See earlier events](https://gnosisscan.io/address/${mechAddress}#events).
```

**Review by oaksprout — CHANGES_REQUESTED** [review]

**Review by oaksprout — APPROVED** [review]

**Comment by oaksprout** [comment]
@mohandast52 looks like once the conflicts are resolved we can merge

**Pull request merged**